integration/sql/impl/src/visitor.rs

```diff
@@ -384,6 +403,14 @@ impl Visit for FunctionArgExpr {
     }
 }
 
+impl Visit for WindowType {
```

Comment:
Wouldn't it be cool to have a unit test for `WindowType` ?

Reply:
Added.